typescript/cli/src/config/chain.ts

```diff
@@ -90,6 +90,52 @@ export async function createChainConfig({
     protocol: ProtocolType.Ethereum,
     rpcUrls: [{ http: rpcUrl }],
   };
+  const wantBlockConfig = await confirm({
```

Comment:
I would hide these behind an advanced question or something. Let's keep it simple in the default case

Reply:
What I understand is instead of directly asking for these configs first the user will be prompted if they want to add advanced configs for this chain, if yes then all these prompts will show up. Please validate my understanding @nambrot